plugins/TestRunner/Commands/SyncScreenshots.php

```diff
@@ -27,7 +27,7 @@ class SyncScreenshots extends ConsoleCommand
      */
     private $logger;
 
-    const buildURL = "https://builds-artifacts.matomo.org";
+    const BUILDURL = "https://builds-artifacts.matomo.org";
```

Comment:
Technically BC, but can be excluded from our public API by defining it being part of the tests suite.

Reply:
we have actually introduced that one when migration from travis to github actions, so it shouldn't be in use by anyone else.

Reply:
In Matomo the way to mark a method/class a public API is to add an `@api` annotation. All those methods will then be published on https://developer.matomo.org/api-reference/classes
Everything that is not part of that list isn't public API and might thus be subject to change. At least that was the way we handled that in the past, don't know if that is clearly documented somewhere.

Reply:
Yeah, we may need to tweak the documentation around this. I guess we can define whatever we want as our public API that we need to conform with SemVer to. At the moment the list is not enough as the classes there are not themselves marked as `@api`, it could be just some methods from those classes.

So I guess we can do all the changes here as long as we cross-check against the current list of `@api` annotated classes and methods, and clarify that that is the form and shape of our SemVer commitment.